src/spanningtrees/kruskal.jl

```diff
@@ -0,0 +1,55 @@
+immutable KruskalHeapEntry{T}
+    edge::Pair{Int, Int}
+    dist::T
+end
+
+isless(e1::KruskalHeapEntry, e2::KruskalHeapEntry) = e1.dist < e2.dist
+
+""" Performs Quick-Find algorithm
+to check for connected vertices
+"""
+function quick_find(nodes, p, q)
+    pid = nodes[p]
+    qid = nodes[q]
+    for i in 1:length(nodes)
+        if nodes[i] == pid
+            nodes[i] = qid
+        end
+    end
+end
```

Comment:
This mutates nodes, so should have a `!` at the end of the function name. Also, the docstring needs a reference to the quick-find algorithm, along with inputs and expected behavior.